tema1/script.js

```diff
@@ -0,0 +1,26 @@
+console.log("EXERCITIUL 1");
+
+var variabila = "a";
```

Comment:
aici o sa te rog sa modifici in ```let```, doarece este standardul modern de la javascript spre deosebire de ```var```. 

De asemenea, cand am mentionat declara variabila a, ma refeream la 
```js
let a;
```
Ceea ce ai facut tu se interpreteaza ca "Declara o variabila numita 'variabila' si apoi atribuie textul 'a' acelei variabile"